tests/run/test_fstring.pyx

```diff
@@ -849,6 +849,14 @@ f'{a * x()}'"""
         self.assertEqual(f'{d["foo"]}', 'bar')
         self.assertEqual(f"{d['foo']}", 'bar')
 
+    def test_inline_compiler_directives(self):
```

Comment:
Well, this isn't quite the right file for testing `cython.inline()`. It actually tests f-strings.
There is a test file dedicated to the inline feature in `Cython/Build/Tests/TestInline.py`.

Reply:
Thanks for clarifying. I've moved the unit test to the appropriate location.